1.8/usage/haproxy.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Configuring HTTP Proxy in Front of DC/OS
+nav_title: Configure HTTP Proxy
+menu_order: 61
+---
+
+You can configure a secure communication channel between HAProxy and the DC/OS Admin Router, using a custom hostname and port.
```

Comment:
Let me try to find an introduction that describes the purpose of this a little more exhaustively...

"""
Sometimes it is required to put an HTTP proxy in front of DC/OS Admin Router. For that setup to work properly, the proxy needs to perform on-the-fly HTTP request and response header modification (the reason for that is that DC/OS is not aware of the custom hostname and port that is being used by user agents to address the HTTP proxy).

Below, you can find a tested HAProxy configuration example that takes care of named request/response rewriting. Furthermore, that configuration example ensures that the communication between HAProxy and DC/OS Admin Router is TLS-encrypted. It uses DC/OS' CA certificate so that HAProxy can verify the TLS certificate presented by Admin Router.
"""